Action.c

```diff
@@ -595,6 +601,24 @@ static Htop_Reaction actionShowLocks(State* st) {
    return HTOP_REFRESH | HTOP_REDRAW_BAR;
 }
 
+#if defined(HAVE_BACKTRACE)
+static Htop_Reaction actionBacktrace(State *st) {
+   const Process* process = (Process*) Panel_getSelected((Panel*)st->mainPanel);
+   if (!process)
+      return HTOP_OK;
+
+   BacktracePanel *panel = BacktracePanel_new(process, st->host->settings);
+   ScreenManager *screenManager = ScreenManager_new(NULL, st->host, st, false);
+   ScreenManager_add(screenManager, (Panel *)panel, 0);
```

Comment:
```suggestion
   ScreenManager_add(screenManager, &panel->super, 0);
```

Reply:
I'm not sure about this one, `(Panel *)panel` and `&panel->super` are equivalent and I don't see much in the code that `super` of `Panel` is used with `ScreenManager_add`.

Moreover, what I want to add to the `ScreenManager` is `BacktracePanel`, not the super panel. In terms of semantic, I prefer `ScreenManager_add(screenManager, (Panel *)panel, 0)`.

Reply:
I'd also recommend doing the typecast in this case. Both for following with other such panels and also to make it clearer that the whole (backtrace) panel structure is passed.